### ISSUE TITLE
Bug: broken fanart.tv integration

### ISSUE THREAD

**Issue by sasspaper** [x6r/mpcrp#3]
I couldn't get this to work at all. It doesn't pull the info from fanart.tv nor does it show the movie name, only the file name. Screenshots bellow. 

![2023-08-17_12-35](https://github.com/fawni/mpcrp/assets/127695532/6ea11d46-25ed-40ca-93fa-21225f399498)
![2023-08-17_12-35_1](https://github.com/fawni/mpcrp/assets/127695532/e06e10dc-8a85-4d70-8220-bcfed4331c35)
![2023-08-17_12-35_2](https://github.com/fawni/mpcrp/assets/127695532/33493fbd-d766-4edb-9743-fe10d4f6f941)

"eg: a video named cat.mp4 will be assumed to be [Cat Planet Cuties](https://fanart.tv/series/173301/cat-planet-cuties/) as it is the first entry on fanart.tv that has a poster and is a tv show or a movie."

https://github.com/fawni/mpcrp/assets/127695532/eb2e8415-2463-40c5-a079-54e960cd38ed

**Comment by fawni** [comment]
yes it is currently broken i'm not really working on it sorry. it uses some really bad practices (nothing unsafe) so you probably should use another project for now. i might rewrite at some point but that's unlikely if i'm being honest.

**Comment by sasspaper** [comment]
Ok! Thanks anyway (=^ ◡ ^=)
